### ISSUE TITLE
1

### ISSUE THREAD

**Issue by raydtaallh** [raydtaallh/Raed#1]
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import accuracy_score

# تحميل البيانات
# تأكد من أن لديك بيانات مناسبة في ملف CSV
data = pd.read_csv('crypto_data.csv')

# تجهيز البيانات
# افترض أن لديك أعمدة مثل 'feature1', 'feature2', ..., 'label'
X = data[['feature1', 'feature2', 'feature3']]
y = data['label']

# تقسيم البيانات إلى تدريب واختبار
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# قياس البيانات
scaler = StandardScaler()
X_train_scaled = scaler.fit_transform(X_train)
X_test_scaled = scaler.transform(X_test)

# إنشاء نموذج الانحدار اللوجيستي
model = LogisticRegression()

# تدريب النموذج
model.fit(X_train_scaled, y_train)

# التنبؤ
y_pred = model.predict(X_test_scaled)

# حساب دقة النموذج
accuracy = accuracy_score(y_test, y_pred)
print(f"Accuracy: {accuracy:.2f}")

# التنبؤ بالبيانات الجديدة
new_data = [[value1, value2, value3]]  # استبدل بالقيم الفعلية
new_data_scaled = scaler.transform(new_data)
prediction = model.predict(new_data_scaled)

print("العملة الرقمية ستتحرك:", "صعودًا" if prediction[0] == 1 else "هبوطًا")
